articles/install-guide/setup/vs-editors/vs-2017.md

```diff
@@ -1,28 +1,41 @@
 ---
-title: Installing the Q# development environment for VS 2017
+title: Installing the Q# development environment for VS 2017 or later
```

Comment:
@bettinaheim made the point earlier that the 0.9 extension will be VS2019 only, so we may want to change the title in preparation for that. My apologies for the confusion!
```suggestion
title: Installing the Q# development environment for VS 2019
```